packages/EasyDecision/src/Bridge/Symfony/DataCollector/DecisionContextDataCollector.php

```diff
@@ -0,0 +1,80 @@
+<?php
+declare(strict_types=1);
+
+namespace EonX\EasyDecision\Bridge\Symfony\DataCollector;
+
+use EonX\EasyDecision\Interfaces\DecisionContextInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\DataCollector\DataCollector;
+
+final class DecisionContextDataCollector extends DataCollector
+{
+    /**
+     * @const string
+     */
+    public const NAME = 'easy_decision.decision_context_collector';
+
+    /**
+     * @var \EonX\EasyDecision\Interfaces\DecisionContextInterface
+     */
+    private $decisionContext;
+
+    public function __construct(DecisionContextInterface $decisionContext)
```

Comment:
The `DecisionContextInterface` isn't defined as a service, and the main reason why it's because there is one context for each decision made. What happen when multiple decisions are made during the same request?